### PR TITLE
ENH: Refer contributors to Neurostars over tumblr for questions 

### LINKED ISSUE #68: Change `tumblr` links for links to `neurostars`
I've noticed that there there are a few links to `tumblr` throughout the website (e.g. [contribute](https://nipy.org/contribute.html); `neurostars` is mentioned in the [help](https://nipy.org/help.html) and in the Discourse instance as the place to ask questions, though.

I've also noticed that the last question on `tumblr` was 2 years ago.

Maybe `tumblr` can be kept for blog posts, but maybe `tumblr` as a place to get help should be changed to `neurostarts` and/or the neuroimaging mailing list? And maybe the latter two could be better distinguished in their purpose when changing this?

### PR DESCRIPTION
**Pull request by jhlegarreta**
- DOC: Add period to mark sentence ending
- DOC: Refer contributors to Neurostars over tumblr for questions

Fixes #68.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Fixes #68.

File: _data/contribute.yml [hunk 1]
```diff
@@ -4,11 +4,11 @@
...
+          question: "How do I add a package?"i
```
Comment (vsoch) [inline _data/contribute.yml:7]:
```suggestion
          question: "How do I add a package?"
```

Comment (jhlegarreta) [reply _data/contribute.yml]: :100: :eyes: Done in  92bbfab. Thanks.

**Comment by vsoch** [comment]
It looks like there is some issue with the contribute yaml, like 9?

**Comment by jhlegarreta** [comment]
> It looks like there is some issue with the contribute yaml, like 9?

Yes; I am unable to find the issue. The only change wrt the previous working version in ab60e22  was to add a period in `</a>.</p>`. Line 9 has not changed.

**Comment by vsoch** [comment]
Let me see if I can take a look.

File: _data/contribute.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@
...
+          answer: "<p>For general questions about methods, analysis, and code, the best place to maximize the number of eyes on your issue is by way of <a href='https://neurostars.org/' target='_blank'>Neurostars</a>. Questions about specific packages are best to post as issues on the repository itself, a link to which you can find on any of the package pages. To be added as a contributor or ask a question about the site, you can reach us <a href='mailto:neuroimaging@python.org'>by email</a>. Although you can ask your question through the <a href="http://neuroimaging.tumblr.com/ask">neuroimaging tumblr ask space</a> note that it gets less attention than the previous resources.</p>"
```
Comment (vsoch) [inline _data/contribute.yml:20]:
```suggestion
          answer: "<p>For general questions about methods, analysis, and code, the best place to maximize the number of eyes on your issue is by way of <a href='https://neurostars.org/' target='_blank'>Neurostars</a>. Questions about specific packages are best to post as issues on the repository itself, a link to which you can find on any of the package pages. To be added as a contributor or ask a question about the site, you can reach us <a href='mailto:neuroimaging@python.org'>by email</a>. Although you can ask your question through the <a href='http://neuroimaging.tumblr.com/ask'>neuroimaging tumblr ask space</a> note that it gets less attention than the previous resources.</p>"
```
You can't have double quotes inside the double quotes to distinguish the field.

Comment (jhlegarreta) [reply _data/contribute.yml]: :man_facepalming: Thanks @vsoch . Changed for simple quotes in a3bdd79.

**Review by vsoch — APPROVED** [review]

**Pull request merged**